### PR TITLE
JavaScript-I completed

### PR DESCRIPTION
**Pull request by perryahern**
Assignment completed

### CODE REVIEW

File: src/objects.js [hunk 1]
```diff
@@ -5,22 +5,50 @@ const keys = (obj) => {
   // Retrieve all the names of the object's properties.
   // Return the keys as strings in an array.
   // Based on http://underscorejs.org/#keys
+  const newArray = [];
+
+  Object.keys(obj).forEach((key, index) => {
+    newArray[index] = key;
```
Comment (sperrye) [inline src/objects.js:11]: Object.keys(obj) returns an array of keys.

**Review by sperrye — COMMENTED** [review]
Great work

File: src/objects.js (hunk 1 above)
Comment (perryahern) [reply src/objects.js:11]: Are you saying I should have done `return Object.keys(obj);` to simplify this?

Comment (sperrye) [reply src/objects.js:11]: Yes, just showing you that you can do it with one step: `return Object.keys(obj)`

**Pull request closed without merging**